### PR TITLE
Added shopping bag to modules

### PR DESCRIPTION
**Pull request by EPuncker**
- Added shopping bag to NPC "buy with backpacks" function as of Tibia 10.55 update
- Changed some commands to meta-table commands

### CODE REVIEW

File: data/npc/lib/npcsystem/modules.lua [hunk 1]
```diff
@@ -1116,7 +1116,7 @@ if Modules == nil then
 					return false
 				end
 				if shop_itemid[cid] == ITEM_PARCEL then
-					doNpcSellItem(cid, ITEM_LABEL, shop_amount[cid], shop_subtype[cid], true, false, 1988)
+					doNpcSellItem(cid, ITEM_LABEL, shop_amount[cid], shop_subtype[cid], true, false, 23782)
```
Comment (DSpeichert) [inline data/npc/lib/npcsystem/modules.lua]: Can you make it `local backpackID = 23782` here as well like the in the function above?
Or just make it a "global" backpackContainerId at the top of the file once for all functions to use.

Comment (EPuncker) [reply data/npc/lib/npcsystem/modules.lua]: like I did now?

Comment (DSpeichert) [reply data/npc/lib/npcsystem/modules.lua]: I think it looks good. Did you test that?

Comment (EPuncker) [reply data/npc/lib/npcsystem/modules.lua]: Just did, and not working, need to edit a few thins in npc.lua as well, I'll sort it out and send a fix

**Comment by EPuncker** [comment]
> Should we add ITEM_SHOPPING_BAG be a const instead of a scripted declared itemid?
> https://github.com/otland/forgottenserver/blob/master/src/const.h#L463

I think so

File: data/npc/lib/npcsystem/modules.lua (hunk 1 above)
Comment (EPuncker) [reply data/npc/lib/npcsystem/modules.lua]: for some odd reason the "buy with backpack" is adding an empty container (without my modified code) and I don't know how to fix it, and seems like nobody reported it until now:

wJJc6bA

if someone is able to confirm and fix it, I'll be grateful, you can even include the content from my PR and close it, I'm not able to fix it myself

Comment (EPuncker) [reply data/npc/lib/npcsystem/modules.lua]: pr #2699 fixes the issue, after it gets approved (I tested and its working), I'll make a new and clean PR with my changes, you can close this one now

**Pull request closed without merging**